### PR TITLE
Font size change on Alex Net

### PR DESCRIPTION
**Pull request by erupturatis**
Pr for #49. I didn't change the fontsize directly because it would lead to the text overflowing the canvas. I used a variable to scale the whole text canvas instead

### CODE REVIEW

**Comment by alexlenail** [comment]
Thanks for the contribution @erupturatis. 
Would you be willing to remove the console.log?

**Comment by erupturatis** [comment]
Forgot about it, sorry. Hope it is fine in rest.

File: AlexNet.html [hunk 1]
```diff
@@ -129,6 +129,10 @@ <h4>Style:</h4>
                                 <input type="color" value="#ffbbbb" id="color3">
                                 <label for="color3">Color 3</label>
                             </div>
+                            <div>
+                                <label for="fontScale">Font Size</label>
+                                <input type="range" id="fontScale" name="" min="1" max="3" step="0.2" value="1" style="position: relative; top: 3px;">
+                            </div>
```
Comment (alexlenail) [inline AlexNet.html]: Could you move this to below "Show Conv Dimensions"

**Review by alexlenail — CHANGES_REQUESTED** [review]

File: AlexNet.html (hunk 1 above)
Comment (erupturatis) [reply AlexNet.html]: Of course

**Pull request merged**